core-js/src/main/javascript/cdf/addIns/template/templateBase.js

```diff
@@ -67,7 +68,7 @@ define([
         type: opt.messages.config.style[type].type || "info",
         icon: opt.messages.config.style[type].icon || "comment"};
       Logger.log(opt.messages.error[message] || "", type);
-      return _.template(opt.messages.config.template)(completeMsg);
+      return _.template(opt.messages.config.template, completeMsg);
```

Comment:
underscore lib was recently updated to version 1.8.3, _.template was changed in [1.7.0](http://underscorejs.org/#1.7.0), _.template always returns a function now, so this change will break the functionality